packages/core/src/sagas/markAllAsSpokenOnStopSpeakActivitySaga.js

```diff
@@ -13,6 +13,7 @@ function* markAllAsSpoken() {
   }
 }
 
+// TODO: [P4] We should turn this into a reducer instead
```

Comment:
file an issue? Or combine with the one above?

Reply:
We need a bot for these.